src/Raven.Server/Documents/Indexes/Persistence/Lucene/IndexOperationBase.cs

```diff
@@ -37,39 +35,43 @@ protected IndexOperationBase(Index index, Logger logger)
             _logger = logger;
         }
 
-        protected static RavenPerFieldAnalyzerWrapper CreateAnalyzer(Func<Analyzer> createDefaultAnalyzer, IndexDefinitionBase indexDefinition, bool forQuerying = false)
+        protected static RavenPerFieldAnalyzerWrapper CreateAnalyzer(Func<LowerCaseKeywordAnalyzer> createDefaultAnalyzer, IndexDefinitionBase indexDefinition, bool forQuerying = false)
```

Comment:
Why this change?

Reply:
Just to be clear what default analyzer actually is. All places that call `CreateAnalyzer` are creating instance of `LowerCaseKeywordAnalyzer`

Reply:
fixed